### ISSUE TITLE
Publish tool tests fail on OSX

### ISSUE THREAD

**Issue by davidfowl** [aspnet/IISIntegration#104]
```
xUnit.net DNX Runner (64-bit osx.10.11-x64)
  Discovering: Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests
  Discovered:  Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests
  Starting:    Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests
    Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.WebConfigTransformFacts.WebConfigTransform_overwrites_processPath [FAIL]
      Assert.Equal() Failure
                  ↓ (pos 2)
      Expected: ..\app.exe
      Actual:   ../app.exe
                  ↑ (pos 2)
      Stack Trace:
           at Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.WebConfigTransformFacts.WebConfigTransform_overwrites_processPath()
      Assert.True() Failure
      Expected: True
      Actual:   False
    Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.WebConfigTransformFacts.WebConfigTransform_overrites_value_for_ASPNET_APPLICATIONBASE [FAIL]
      Stack Trace:
           at Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.WebConfigTransformFacts.WebConfigTransform_overrites_value_for_ASPNET_APPLICATIONBASE()
      Assert.True() Failure
      Expected: True
      Actual:   False
    Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.WebConfigTransformFacts.WebConfigTransform_creates_new_config_if_one_has_unexpected_format [FAIL]
      Stack Trace:
           at Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.WebConfigTransformFacts.WebConfigTransform_creates_new_config_if_one_has_unexpected_format()
      Assert.True() Failure
    Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.WebConfigTransformFacts.WebConfigTransform_creates_new_config_if_one_does_not_exist [FAIL]
      Expected: True
      Actual:   False
      Stack Trace:
           at Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.WebConfigTransformFacts.WebConfigTransform_creates_new_config_if_one_does_not_exist()
      Assert.True() Failure
      Expected: True
    Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.WebConfigTransformFacts.WebConfigTransform_fixes_aspnetcore_casing [FAIL]
      Actual:   False
      Stack Trace:
           at Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.WebConfigTransformFacts.WebConfigTransform_fixes_aspnetcore_casing()
      Assert.True() Failure
      Expected: True
    Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.WebConfigTransformFacts.WebConfigTransform_adds_missing_elements(elementNames: ["system.webServer"]) [FAIL]
      Actual:   False
      Stack Trace:
           at Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.WebConfigTransformFacts.WebConfigTransform_adds_missing_elements(String[] elementNames)
      Assert.True() Failure
      Expected: True
      Actual:   False
    Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.WebConfigTransformFacts.WebConfigTransform_adds_missing_elements(elementNames: ["add"]) [FAIL]
      Stack Trace:
           at Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.WebConfigTransformFacts.WebConfigTransform_adds_missing_elements(String[] elementNames)
      Assert.True() Failure
      Expected: True
      Actual:   False
    Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.WebConfigTransformFacts.WebConfigTransform_adds_missing_elements(elementNames: ["handlers"]) [FAIL]
      Stack Trace:
           at Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.WebConfigTransformFacts.WebConfigTransform_adds_missing_elements(String[] elementNames)
    Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.WebConfigTransformFacts.WebConfigTransform_adds_missing_elements(elementNames: ["aspNetCore"]) [FAIL]
      Assert.True() Failure
      Expected: True
      Actual:   False
      Stack Trace:
           at Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.WebConfigTransformFacts.WebConfigTransform_adds_missing_elements(String[] elementNames)
      Assert.True() Failure
      Expected: True
      Actual:   False
    Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.WebConfigTransformFacts.WebConfigTransform_adds_missing_elements(elementNames: ["environmentVariables"]) [FAIL]
      Stack Trace:
           at Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.WebConfigTransformFacts.WebConfigTransform_adds_missing_elements(String[] elementNames)
      Assert.True() Failure
      Expected: True
      Actual:   False
    Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.WebConfigTransformFacts.WebConfigTransform_adds_missing_elements(elementNames: ["environmentVariable"]) [FAIL]
      Stack Trace:
           at Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.WebConfigTransformFacts.WebConfigTransform_adds_missing_elements(String[] elementNames)
      Assert.True() Failure
      Expected: True
      Actual:   False
      Stack Trace:
    Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.WebConfigTransformFacts.WebConfigTransform_adds_missing_elements(elementNames: ["handlers", "aspNetCore", "environmentVariables"]) [FAIL]
           at Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.WebConfigTransformFacts.WebConfigTransform_adds_missing_elements(String[] elementNames)
      Assert.True() Failure
      Expected: True
    Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.WebConfigTransformFacts.WebConfigTransform_correctly_configures_for_Azure [FAIL]
      Actual:   False
      Stack Trace:
           at Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.WebConfigTransformFacts.WebConfigTransform_correctly_configures_for_Azure()
No web.config found. Creating '/var/folders/bj/8g_yfry17nn_1ff492j_4m5w0000gn/T/8410b21e-c0b2-4194-a96b-3a9096419f7a/publishOut/mywebroot/web.config'
      Assert.Equal() Failure
                  ↓ (pos 2)
    Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.PublishIISCommandFacts.PublishIIS_webroot_switch_takes_precedence_over_hosting_json [FAIL]
      Expected: ..\projectDir.exe
      Actual:   ../projectDir.exe
                  ↑ (pos 2)
      Stack Trace:
           at Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.PublishIISCommandFacts.PublishIIS_webroot_switch_takes_precedence_over_hosting_json()
No web.config found. Creating '/var/folders/bj/8g_yfry17nn_1ff492j_4m5w0000gn/T/566cf6e5-b22a-4c8d-a098-b81a851dab6c/publishOut/wwwroot/web.config'
      Assert.Equal() Failure
                  ↓ (pos 2)
      Expected: ..\awesomeApp.exe
      Actual:   ../awesomeApp.exe
                  ↑ (pos 2)
      Stack Trace:
           at Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.PublishIISCommandFacts.PublishIIS_reads_application_name_from_project_json_if_exists(String projectName)
    Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.PublishIISCommandFacts.PublishIIS_reads_application_name_from_project_json_if_exists(projectName: "awesomeApp") [FAIL]
No web.config found. Creating '/var/folders/bj/8g_yfry17nn_1ff492j_4m5w0000gn/T/819d06de-1b3f-4f70-9212-e6252c5238ba/publishOut/wwwroot/web.config'
      Assert.Equal() Failure
    Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.PublishIISCommandFacts.PublishIIS_reads_application_name_from_project_json_if_exists(projectName: "awesome.App") [FAIL]
                  ↓ (pos 2)
      Expected: ..\awesome.App.exe
      Actual:   ../awesome.App.exe
                  ↑ (pos 2)
      Stack Trace:
           at Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.PublishIISCommandFacts.PublishIIS_reads_application_name_from_project_json_if_exists(String projectName)
No web.config found. Creating '/var/folders/bj/8g_yfry17nn_1ff492j_4m5w0000gn/T/5bffe8f0-0754-4657-bed7-149465ac058d/publishOut/mywebroot/web.config'
      Assert.Equal() Failure
                  ↓ (pos 2)
      Expected: ..\projectDir.exe
      Actual:   ../projectDir.exe
                  ↑ (pos 2)
      Stack Trace:
           at Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.PublishIISCommandFacts.PublishIIS_uses_webroot_from_hosting_json()
    Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.PublishIISCommandFacts.PublishIIS_uses_webroot_from_hosting_json [FAIL]
No web.config found. Creating '/var/folders/bj/8g_yfry17nn_1ff492j_4m5w0000gn/T/ef339ef6-edc8-47f8-ad18-2448e0b6bba8/publishOut/wwwroot/web.config'
    Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.PublishIISCommandFacts.PublishIIS_accepts_path_to_project_json_as_project_path(projectDir: "projectDir") [FAIL]
      Assert.Equal() Failure
                  ↓ (pos 2)
      Expected: ..\projectDir.exe
      Actual:   ../projectDir.exe
                  ↑ (pos 2)
      Stack Trace:
           at Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.PublishIISCommandFacts.PublishIIS_accepts_path_to_project_json_as_project_path(String projectDir)
No web.config found. Creating '/var/folders/bj/8g_yfry17nn_1ff492j_4m5w0000gn/T/da2ce0d7-ff75-4a2c-99b4-cb9ce7b0ba09/publishOut/wwwroot/web.config'
      Assert.Equal() Failure
    Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.PublishIISCommandFacts.PublishIIS_accepts_path_to_project_json_as_project_path(projectDir: "project.Dir") [FAIL]
                  ↓ (pos 2)
      Expected: ..\project.Dir.exe
      Actual:   ../project.Dir.exe
                  ↑ (pos 2)
      Stack Trace:
           at Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.PublishIISCommandFacts.PublishIIS_accepts_path_to_project_json_as_project_path(String projectDir)
No web.config found. Creating '/var/folders/bj/8g_yfry17nn_1ff492j_4m5w0000gn/T/755ff12d-af52-4412-b05e-f4c6229c8c55/publishOut/wwwroot/web.config'
      Assert.Equal() Failure
    Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.PublishIISCommandFacts.PublishIIS_uses_default_values_if_options_not_specified [FAIL]
                  ↓ (pos 2)
      Expected: ..\projectDir.exe
      Actual:   ../projectDir.exe
                  ↑ (pos 2)
      Stack Trace:
           at Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.PublishIISCommandFacts.PublishIIS_uses_default_values_if_options_not_specified()
Updating web.config at '/var/folders/bj/8g_yfry17nn_1ff492j_4m5w0000gn/T/122abc49-cf91-4d8b-be8e-bc83078dfc4b/publishOut/wwwroot/web.config'
      Assert.Equal() Failure
    Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.PublishIISCommandFacts.PublishIIS_modifies_existing_web_config [FAIL]
                  ↓ (pos 2)
      Expected: ..\projectDir.exe
      Actual:   ../projectDir.exe
                  ↑ (pos 2)
      Stack Trace:
           at Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests.PublishIISCommandFacts.PublishIIS_modifies_existing_web_config()
  Finished:    Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests
=== TEST EXECUTION SUMMARY ===
   Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests  Total: 37, Errors: 0, Failed: 21, Skipped: 0, Time: 0.413s
```

/cc @moozzyk does this matter?

**Comment by moozzyk** [comment]
@davidfowl it doesn't but it shouldn't fail. I will take a look at this.

**Comment by moozzyk** [comment]
This is not a test issue. We need to update code for this - dotnet can't handle forward slashes on Windows (https://github.com/dotnet/cli/issues/2373) and AspNetCoreModule crashes on forward slashes in the `processPath` attribute (#127).

**Comment by moozzyk** [comment]
Fixed in 7f7742577d4a65aebd180277d7c0790d6c2e0207
